src/main/java/com/bugsnag/android/BreadcrumbType.java

```diff
@@ -0,0 +1,48 @@
+package com.bugsnag.android;
+
+/**
+ * Recognized types of breadcrumbs
+ */
+public enum BreadcrumbType {
+    /**
+     * An error was sent to Bugsnag (internal use only)
+     */
+    ERROR ("error"),
+    /**
+     * A log message
+     */
+    LOG ("log"),
+    /**
+     * A manual invocation of `leaveBreadcrumb` (default)
+     */
+    MANUAL ("manual"),
+    /**
+     * A navigation event, such as a window opening or closing
+     */
+    NAVIGATION ("navigation"),
+    /**
+     * A background process such as a database query
+     */
+    PROCESS ("process"),
+    /**
+     * A network request
+     */
+    REQUEST ("request"),
+    /**
+     * A change in application state, such as launch or memory warning
+     */
+    STATE ("state"),
+    /**
+     * A user action, such as tapping a button
+     */
+    USER ("user");
+
+    private final String type;
+
+    BreadcrumbType(String type) {
+        this.type = type;
+    }
+
+    String serialize() { return type; }
```

Comment:
Did you make it public? If you override toString you need to make it public otherwise it will be trying to reduce the access privileges of an inherited method which is not allowed.